src/main/java/com/toptrumps/core/Game.java

```diff
@@ -1,4 +1,23 @@
 package com.toptrumps.core;
 
+import java.util.ArrayList;
+
 public class Game {
-}
+
+    private ArrayList<Player> players;
+
+    public Game() {
+        players = new ArrayList<Player>();
```

Comment:
Maybe a better idea is to make the constructor from the game absorb the number of players! Think with me: we NEED to know the number of players to start the game, don't we?